src/__tests__/parser.test.ts

```diff
@@ -121,19 +121,29 @@ describe("Parsing CSVs ", () => {
       warningUndefinedDecimals,
       warningInvalidTokenAddress,
       warningInvalidReceiverAddress,
-    ] = warnings.map((warning) => warning.message);
+    ] = warnings;
     expect(payment).to.be.empty;
 
-    expect(warningNegativeAmount).to.equal(
-      "1: Only positive amounts possible: -1"
+    expect(warningNegativeAmount.message).to.equal(
+      "Only positive amounts possible: -1"
```

Comment:
What is the -1 for?

Reply:
The idea is, that the warning contains the invalid amount, which was provided.
And this testcase provides a "-1" as the amount to transfer.